### PR TITLE
feat(dev): Run bootstrap script when dev env changes happens

### PR DESCRIPTION
**Pull request by armenzg**
This will help catch regressions on the script that new engineers execute.

### CODE REVIEW

**Comment by armenzg** [comment]
@billyvg what are your thoughts about this? Should we run this on a schedule? or every time we touch something that could regress it? Both?

A schedule could catch issues when we don't make any code changes for few weeks and a PR could catch issues when we touched those files.

**Comment by billyvg** [comment]
I think this makes sense, I might prefer the scheduled to run in the bootstrap repo? I wonder if there's an easy way to subscribe to failures for a specific workflow...

**Comment by armenzg** [comment]
> I think this makes sense, I might prefer the scheduled to run in the bootstrap repo? I wonder if there's an easy way to subscribe to failures for a specific workflow...

It's the dumbest system. If I make changes to the workflow *I* only get notified. If you or someone else then enables/disable the workflow *they* get notified instead of me. Last person touching the workflow becomes the person that gets email.

We might need a GH action or GH app to route status notifications for scheduled workflows.

**Comment by billyvg** [comment]
Hm that is pretty dumb

**Review by billyvg — APPROVED** [review]

File: .github/workflows/bootstrap.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: bootstrap
+on:
+  schedule:
+    # Run on Fridays & Mondays in hopes to fix things before new engineers run the script
```
Comment (armenzg) [inline .github/workflows/bootstrap.yml:4]:
```suggestion
    # Run on Mondays in hopes to fix things before new engineers run the script
```

**Pull request merged**